students/ShinTran/lesson04/kata.py

```diff
@@ -0,0 +1,78 @@
+'''
+Shin Tran
+Python 210
+Lesson 4 Assignment
+'''
+
+
+import string # For punctuation
+import random # For a random number generator
+
+# To store all the two word combo as keys and the following word as a value
+word_dict = {}
+
+
+# Opens a text file, parses though it line by line, storing two consecutive
+# words as keys, and the following word as a value
+def open_file():
+	global word_dict
```

Comment:
it is best to avoid using `global` unless really needed.

in this case your function could return `word_dict` value instead of using it as a global.